MANIFEST.in

```diff
@@ -1,5 +1,5 @@
-include pytorch_lightning/py.typed  # marker file for PEP 561
-include CHANGELOG.md
-recursive-include requirements *.txt
-include requirements.txt
+exclude *.toml  # project config
+exclude requirements.txt
```

Comment:
It's a good idea to distribute the requirements. Why remove it? Also it needs to be done recursively.